oaknational/Oak-Web-Application
# [LESQ-46] Feat/connect search to hasura

## Description

connect teachers-2023/search to hasura endpoint

## Issue(s)

Fixes #

## How to test

1. Go to beta/teachers-2023/search
3. You should see filters and page working as before but with 2023 data

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist
